registry/storage/driver/s3-aws/s3.go

```diff
@@ -153,6 +154,11 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		return nil, fmt.Errorf("No bucket parameter provided")
 	}
 
+	regionEndpoint, ok := parameters["regionendpoint"]
```

Comment:
Sorry I actually just noticed this, but this should follow the pattern of `nil` checking like the other parameters, otherwise it's possible for `fmt.Sprint(regionEndpoint)` to return `"<nil>"`. Other than that this looks good to me